File: 5kyu/pigLatin.py
# https://www.codewars.com/kata/520b9d2ad5c005041100000f

def pig_it(text):
    s = text.split(" ")
    n = []
    for word in s:
        if word.isalpha():
            if len(word) == 1:
                n.append(f"{word}ay")
            else:
                n.append(f"{word[1:]}{word[0]}ay")
        else:
            n.append(word)
    return ' '.join(n)

File: src/5kyu/pigLatin.js
// https://www.codewars.com/kata/520b9d2ad5c005041100000f

const pigIt = str => {
    let words = str.split(' ').map(word=>word.split(''))
    for (word of words) {
      if (/[a-zA-Z]/.test(word)) {
        let firstLetter = word.shift()
        word.push(firstLetter,'a','y')
      }
    }
    return words.map(word=>word.join('')).join(' ')
  }